inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/tubemq/TubeMQSourceOperator.java

```diff
@@ -110,17 +110,22 @@ public Map<String, List<StreamSource>> getSourcesMap(InlongGroupInfo groupInfo,
             tubeMQSource.setTopic(streamInfo.getMqResource());
             tubeMQSource.setGroupId(streamId);
             tubeMQSource.setMasterRpc(masterRpc);
-            if (StringUtils.isNotBlank(streamInfo.getDataType())) {
-                String serializationType = DataTypeEnum.forType(streamInfo.getDataType()).getType();
-                tubeMQSource.setSerializationType(serializationType);
-            }
             tubeMQSource.setIgnoreParseError(streamInfo.getIgnoreParseError());
 
             for (StreamSource sourceInfo : streamSources) {
                 if (!Objects.equals(streamId, sourceInfo.getInlongStreamId())) {
                     continue;
                 }
-                tubeMQSource.setSerializationType(sourceInfo.getSerializationType());
+
+                if (sourceInfo.getSourceType().equalsIgnoreCase(SourceType.FILE) ||
+                        sourceInfo.getSourceType().equalsIgnoreCase(SourceType.AUTO_PUSH)) {
+                    if (StringUtils.isNotBlank(streamInfo.getDataType())) {
```

Comment:
And also, can we extract those common codes into `AbstractSourceOperator`?

Reply:
Done